_Group Projects/Group_2/Week_8/program.do

```diff
@@ -0,0 +1,74 @@
+cap program drop pmt 
+clear all
+
+program define pmt, rclass // define an r-class program so that the program writes into return and we can get stuff out of the program
+
+// generate the cutoff for the bottom quintile of 30 observations
+set obs 18 // our intervention has 18 strata 
+gen cluster_id = _n 
```

Comment:
I wondered why you named your strata "cluster" if they're strata? (For what it's worth, I find the difference between the two a bit confusing and think we actually misnamed them in our files; sharing some resources Ben passed on to learn about clusters and clustered standard errors below. For fixed effects, @yashdhuldhoya , this also gets into the conversation we've had in Quant III about using fixed effects when our rates of treatment are different across groups (e.g., schools), and the fixed effects convo we had this week and will be having next week--I found it helpful to think of it in that context)

- [Abadie et al., When Should You Adjust Standard Errors for Clustering?](https://economics.mit.edu/sites/default/files/2022-09/When%20Should%20You%20Adjust%20Standard%20Errors%20for%20Clustering.pdf)
- [World Bank Blogs, When should you cluster standard errors? New wisdom from the econometrics oracle](https://blogs.worldbank.org/impactevaluations/when-should-you-cluster-standard-errors-new-wisdom-econometrics-oracle)

Reply:
This is super helpful! Thanks Sylvia :)